lib/model/query/submissions.js

```diff
@@ -8,7 +8,7 @@
 // except according to the terms contained in the LICENSE file.
 
 const config = require('config');
-const { map } = require('ramda');
+const { always, identity, ifElse, map, pick, without, ...R } = require('ramda');
```

Comment:
:nit 🙃

How about 
```suggestion
const { always, identity, ifElse, map, pick, without, equals: rEquals } = require('ramda');
```

and then on line # 347

```diff
- const ExportSubmissionDef = cloneFrame(Submission.Def, map(ifElse(R.equals('xml'), always(xmlUnlessEncrypted), identity)));
+ const ExportSubmissionDef = cloneFrame(Submission.Def, map(ifElse(rEquals('xml'), always(xmlUnlessEncrypted), identity)));
```

Reply:
Addressed in https://github.com/getodk/central-backend/pull/1258